pulp_smash/pulp3/pytest_plugin/__init__.py

```diff
@@ -0,0 +1,236 @@
+import asyncio
+import threading
+import socket
+import ssl
+
+from dataclasses import dataclass
```

Comment:
This breaks compatibility with Python 3.6

Reply:
Hmmm, maybe I need to rework this so it doesn't break old release branches....

Reply:
This got addressed https://github.com/pulp/pulp-smash/commit/a04c4ee6626876c385010def5afb488910fce883